Droid/Fragments/CrimeFragment.cs

```diff
@@ -85,8 +85,10 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
             _suspectButton.Click += SuspectButtonClicked;
             _photoButton.Click += PhotoButtonClicked;
             _photoImageView.Click += ImageViewClicked;
-            UpdatePhotoView();
 
+            var viewTreeObserver = _photoImageView.ViewTreeObserver;
```

Comment:
Let's try to create and extension on `View` that does this by taking only an `Action` as parameter. You can call it `DoOnGlobalLayout`.